### ISSUE TITLE
If example.com got hacked, do I need to change master password?

### ISSUE THREAD

**Issue by edouard-lopez** [lesspass/pure#5]
## Question

> If [gmail.com](gmail.com) got hacked, and I need to change password. I need to change master password, right? - Matthew

### TL;DR
Nope, just **change the _counter_** in the _advanced option_ section (see screenshot below).

### Explanation

Your _master password_ is unique to you and used to compute a site-specific password (see #4 for details). 

You should not change it otherwise you will have to remember so many passwords that the app will be useless. You pick one super strong _master password_ and then adding _login_ and _site_ to the recipe make the result unique and strong.

if Gmail got hacked you just need to change the _counter_ in the _advanced option_ section (see screenshot below). By default the counter is set to `1`, so just increment the value.

**N.B.:** Future versions should add features to help manage this scenario.

![lesspass-addon-update-counter](https://cloud.githubusercontent.com/assets/1212392/16683956/999825a6-4502-11e6-9495-556b461a9924.png)
